### PR TITLE
Blueshield Part II: The MOD suit awakening (+ buffs security modsuit armor)

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Security modsuit armor got buffed to be identical to the blueshield armor.

image

Removes the jetpack, upgraded cell, and projectile dampener from the blueshield modsuit and gives it back. Also sets its complexity to default capacity (15 complexity)

## Why It's Good For The Game

Upon review of the security modsuit, it's extremely weak and barely touched when all the other armor overrides buff everything else significantly. There should be a little more of a fighting chance inside space for security.

Also the blueshield modsuit is a great feature, it just needed some balancing to focus it more around saving people from space. It's stripped down roundstart, but it has 15 complexity.

## Changelog

:cl: The Sharkening
balance: Nerfs the blueshield modsuit and gives it back
balance: Security modsuit buffed (	melee = 40, bullet = 30, laser = 30, energy = 40, bomb = 50, bio = 100, fire = 100, acid = 100, wound = 10
/:cl:

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
Might as well use #756 too for balance feedback. Since this is going to now be a trio of PRs as I slowly reintroduce skyrat content back into our server.

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
 	applied_modules = list(
-		/obj/item/mod/module/storage/large_capacity,
+		/obj/item/mod/module/storage, // Bubberstation Edit - Original: Extended storage
 		/obj/item/mod/module/magnetic_harness,
 		/obj/item/mod/module/flashlight,
-		/obj/item/mod/module/jetpack,
-		/obj/item/mod/module/projectile_dampener,
+		// /obj/item/mod/module/jetpack, // Bubberstation Removal
```
Comment (nevimer) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:11]: If the statement is that you wanted blueshields to be good at rescuing in space, they should have this module.

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm [hunk 2]
```diff
@@ -3,15 +3,15 @@
 	icon = 'modular_skyrat/modules/blueshield/icons/praetorian.dmi'
 	icon_state = "praetorian-control"
 	theme = /datum/mod_theme/blueshield
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
 	applied_modules = list(
-		/obj/item/mod/module/storage/large_capacity,
+		/obj/item/mod/module/storage, // Bubberstation Edit - Original: Extended storage
```
Comment (nevimer) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:8]: Brings it down to non-command levels?

Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:8]: Yes

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm (hunk 1 above)
Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:11]: I'll lower the EVA armor then

**Comment by StrangeWeirdKitten** [comment]
image

These are the new values. I.E, I lowered the blueshield modsuit and I've decreased the wound armor on both

File: modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm [hunk 3]
```diff
@@ -15,7 +15,7 @@
 	atom_flags = PREVENT_CONTENTS_EXPLOSION_1
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	siemens_coefficient = 0
-	complexity_max = DEFAULT_MAX_COMPLEXITY + 3
+	complexity_max = DEFAULT_MAX_COMPLEXITY // Bubberstation edit: original 18
```
Comment (Majkl-J) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm:18]:
```suggestion
	complexity_max = DEFAULT_MAX_COMPLEXITY // Bubberstation edit, original: DEFAULT_MAX_COMPLEXITY + 3
```

Comment (Majkl-J) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm:18]: Also can be made modular

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm (hunk 3 above)
Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm:18]: > Also can be made modular

I'll refactor this if I make more modsuit overrides then just the blueshield

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm [hunk 4]
```diff
@@ -3,13 +3,13 @@
 	icon = 'modular_skyrat/modules/blueshield/icons/praetorian.dmi'
 	icon_state = "praetorian-control"
 	theme = /datum/mod_theme/blueshield
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:6]: do they really need that?

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm [hunk 5]
```diff
@@ -3,13 +3,13 @@
...
 	applied_modules = list(
-		/obj/item/mod/module/storage/large_capacity,
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:8]: eeehhh they can affor d to keep it, they can buy the module anyway

File: modular_zubbers/code/datums/armor_overrides.dm [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+/datum/armor/mod_theme_blueshield // Not really great for fighting.
+	melee = 30
+	bullet = 20
+	laser = 20
+	energy = 30
+	bomb = 50
+	bio = 100
+	fire = 100
+	acid = 100
+	wound = 10
```
Comment (projectkepler-ru) [inline modular_zubbers/code/datums/armor_overrides.dm:29]:
```suggestion
	wound = 20
```
could give it some more wound armor while at it to compensate for the reduced armor on everything else.

File: modular_zubbers/code/datums/armor_overrides.dm [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+// Blueshield Armor
+/datum/mod_theme/blueshield
+	armor_type = /datum/armor/mod_theme_blueshield
+
+/datum/armor/mod_theme_blueshield // Not really great for fighting.
+	melee = 30
```
Comment (projectkepler-ru) [inline modular_zubbers/code/datums/armor_overrides.dm:21]:
```suggestion
	melee = 40
```
we can afford to have it at 40 while being worse for  ranged combat (ie laser/bullet)

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm [hunk 8]
```diff
@@ -3,13 +3,13 @@
...
+		/obj/item/mod/module/storage, // Bubberstation Edit - Original: Extended storage
 		/obj/item/mod/module/magnetic_harness,
 		/obj/item/mod/module/flashlight,
 		/obj/item/mod/module/jetpack,
-		/obj/item/mod/module/projectile_dampener,
+		// /obj/item/mod/module/projectile_dampener, Bubberstation Removal
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:12]: Good one, it wasn't really necessary anyway

**Review by projectkepler-ru — APPROVED** [review]

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm (hunk 4 above)
Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:6]: High is 10 MJ. Super is 20 MJ. They can have a stock cell

File: modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm (hunk 5 above)
Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm:8]: They can just buy it.

File: modular_zubbers/code/datums/armor_overrides.dm (hunk 7 above)
Comment (StrangeWeirdKitten) [reply modular_zubbers/code/datums/armor_overrides.dm]: Agreed

**Comment by StrangeWeirdKitten** [comment]
> **This is fine, I guess. ** I just hate the storage shrinking. It'd tedious to deal with fake balance like smaller storage. 

This is something probally addressed in a later PR

**Pull request merged**